src/JXGBoard.jsx

```diff
@@ -539,109 +241,11 @@ export default class JXGBoard extends React.Component {
-                gLine2OffsetXInitial: this.props.gLine2OffsetXInitial,
-                gLine2OffsetYInitial: this.props.gLine2OffsetYInitial,
-                gLine3OffsetX: this.props.gLine3OffsetX,
-                gLine3OffsetY: this.props.gLine3OffsetY,
-                gLine3OffsetXInitial: this.props.gLine3OffsetXInitial,
-                gLine3OffsetYInitial: this.props.gLine3OffsetYInitial,
-                gLine4OffsetX: this.props.gLine4OffsetX,
-                gLine4OffsetY: this.props.gLine4OffsetY,
-                gLine4OffsetXInitial: this.props.gLine4OffsetXInitial,
-                gLine4OffsetYInitial: this.props.gLine4OffsetYInitial,
-                gLine1Dashed: this.props.gLine1Dashed,
-                gLine2Dashed: this.props.gLine2Dashed,
-                gLine3Dashed: this.props.gLine3Dashed,
-                gAlpha: this.props.gAlpha,
-                gA1: this.props.gA1,
-                gA1Initial: this.props.gA1Initial,
-                gA2: this.props.gA2,
-                gA2Initial: this.props.gA2Initial,
-                gA3: this.props.gA3,
-                gA3Initial: this.props.gA3Initial,
-                gA4: this.props.gA4,
-                gA4Initial: this.props.gA4Initial,
-                gA5: this.props.gA5,
-                gA5Initial: this.props.gA5Initial,
-                gA: this.props.gA,
-                gK: this.props.gK,
-                gR: this.props.gR,
-                gY1: this.props.gY1,
-                gY2: this.props.gY2,
-                gCobbDouglasA: this.props.gCobbDouglasA,
-                gCobbDouglasAInitial: this.props.gCobbDouglasAInitial,
-                gCobbDouglasAName: this.props.gCobbDouglasAName,
-                gCobbDouglasL: this.props.gCobbDouglasL,
-                gCobbDouglasLInitial: this.props.gCobbDouglasLInitial,
-                gCobbDouglasLName: this.props.gCobbDouglasLName,
-                gCobbDouglasK: this.props.gCobbDouglasK,
-                gCobbDouglasKInitial: this.props.gCobbDouglasKInitial,
-                gCobbDouglasKName: this.props.gCobbDouglasKName,
-                gCobbDouglasAlpha: this.props.gCobbDouglasAlpha,
-                gCobbDouglasAlphaInitial: this.props.gCobbDouglasAlphaInitial,
-                gNName: this.props.gNName,
-                gFunctionChoice: this.props.gFunctionChoice,
-                gAreaConfiguration: this.props.gAreaConfiguration,
-                gAreaConfigurationInitial: this.props.gAreaConfigurationInitial,
-                gIsAreaDisplayed: this.props.gIsAreaDisplayed,
-                gAreaAName: this.props.gAreaAName,
-                gAreaBName: this.props.gAreaBName,
-                gAreaCName: this.props.gAreaCName,
-
-                gNeedsSubmit: this.props.gNeedsSubmit,
-
                 l1SubmissionOffset: getL1SubmissionOffset(this.props.submission),
                 l2SubmissionOffset: getL2SubmissionOffset(this.props.submission),
                 submission: this.props.submission,
```

Comment:
This line can be removed as the unprocessed `submission` value is included in in `...this.props` below.